internal/api/authz/instance.go

```diff
@@ -66,6 +67,10 @@ func (i *instance) DefaultOrganisationID() string {
 	return i.orgID
 }
 
+func (i *instance) SecurityPolicyAllowedOrigins() []string {
+	return nil
```

Comment:
Not sure, but I did not find this method doing something somewhere? Shouldn't this return something?

Reply:
this function is part of the authz.Instance interface and is used in the SecurityHeaders ServeHTTP function
the mock is currently not needed for that part, but must implement the whole interface